app/models/cherrypick_task.rb

```diff
@@ -33,23 +33,23 @@ def control_positions(batch_id, num_plate, total_wells, num_control_wells, wells
 
     raise StandardError, 'More controls than free wells' if num_control_wells > total_available_positions
 
-    quotient = batch_id
-    size_region = (total_available_positions / num_control_wells)
-    regions = (wells_to_leave_free..(total_wells - 1)).each_slice(size_region).to_a
+    available_positions = (wells_to_leave_free..(total_wells - 1)).to_a
+    # If num plate is equal to the available positions, the cycle is going to be repeated.
```

Comment:
`(wells_to_leave_free..(total_wells - 1))` can just be `(wells_to_leave_free...total_wells)`

Reply:
Ok, changing